source/connect/general-connection.txt

```diff
@@ -11,3 +11,80 @@ General Connection
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`General` connection tab allows you to select a 
+:guilabel:`Connection String Scheme` for your |compass-short| deployment. 
```

Comment:
I think we might be able to be more specific about how the configurations are used to connect to MongoDB, how's something like this?
```suggestion
:guilabel:`Connection String Scheme` and configure the hostname or hosts used to connect to your MongoDB deployment. 
```

Reply:
Done.